util/util.go

```diff
@@ -45,6 +46,16 @@ func Contains[T comparable](haystack []T, needle T) bool {
 	return false
 }
 
+// ContainsContains returns true if any element of haystack .Contains(needle).
+func ContainsContains[T interface{ Contains(U) bool }, U any](haystack []T, needle U) bool {
```

Comment:
1. This is wild. I didn't know Go generics can do that.
2. This is oddly named
3. This is incomprehensible and very against my KISS rules (keep it simple)

Can you express this function in an easier way. It doesn't have to be generic if it is only used once, maybe? Maybe just `ContainsIP`?

Reply:
Changed to non-generic ContainsIP